### PR TITLE
Insect phobia

### PR DESCRIPTION
**Pull request by RaveRadbury**
## About The Pull Request
Adds an insect phobia. It applies to moths, flies, cockroaches, and bees

## Why It's Good For The Game
Nearly every other species, especially roundstart ones, has a fear associated with it. Moths should not be an exception.

## Changelog
:cl:
tweak: New insect phobia
/:cl:

### CODE REVIEW

**Comment by EOBGames** [comment]
Does this apply to bees too?

**Comment by Fikou** [comment]
does this apply to l*zards too?

**Comment by RaveRadbury** [comment]
While we're here, if anyone has thoughts on how to change the method of applying phobias through the quirk system, that's something I could use assistance on. Currently it uses a timer that expires before the round is set up, so phobias only apply correctly if you late join.

**Comment by RaveRadbury** [comment]
> Does this apply to bees too?

brb adding bee content

**Comment by Kaffe-work** [comment]
flypersons?

**Comment by RaveRadbury** [comment]
> flypersons?

` "insects" = typecacheof(list(/datum/species/fly, /datum/species/moth))`

**Comment by Jared-Fogle** [comment]
If it's not too much effort, it'd be funny if hearing speech in moffic procced it too.

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -140,7 +142,9 @@ SUBSYSTEM_DEF(traumas)
 					   "anime" = typecacheof(list(/obj/item/clothing/under/costume/schoolgirl, /obj/item/katana, /obj/item/reagent_containers/food/snacks/sashimi, /obj/item/reagent_containers/food/snacks/chawanmushi,
 					   /obj/item/reagent_containers/food/drinks/bottle/sake, /obj/item/throwing_star, /obj/item/clothing/head/kitty/genuine, /obj/item/clothing/suit/space/space_ninja,
 					   /obj/item/clothing/mask/gas/space_ninja, /obj/item/clothing/shoes/space_ninja, /obj/item/clothing/gloves/space_ninja, /obj/item/vibro_weapon,
-					   /obj/item/nullrod/scythe/vibro, /obj/item/energy_katana, /obj/item/toy/katana, /obj/item/nullrod/claymore/katana, /obj/structure/window/paperframe, /obj/structure/mineral_door/paperframe))
+					   /obj/item/nullrod/scythe/vibro, /obj/item/energy_katana, /obj/item/toy/katana, /obj/item/nullrod/claymore/katana, /obj/structure/window/paperframe, /obj/structure/mineral_door/paperframe)),
+
+						"insects" = typecacheof(list(/obj/item/toy/plush/moth, /obj/item/toy/plush/beeplushie, /obj/item/clothing/mask/rat/bee, /obj/item/clothing/suit/hooded/bee_costume, /obj/structure/beebox))
```
Comment (ATH1909) [inline code/controllers/subsystem/traumas.dm:147]: what about bee suit hoods?

Comment (Fikou) [reply code/controllers/subsystem/traumas.dm:147]: you need a bee costume to see the hood

**Review by Rohesie — APPROVED** [review]

**Pull request merged**